<launch>
  <arg name="bag_path" description="Path to rosbag file or directory"/>
  <arg name="config_file" default="$(find-pkg-share autoware_planning_data_analyzer)/config/planning_data_analyzer.param.yaml" description="Path to parameter file"/>
  <arg name="output_dir" description="Output directory for evaluation results (overrides config file if specified)"/>

  <node pkg="autoware_planning_data_analyzer" exec="autoware_planning_data_analyzer_node" name="autoware_planning_data_analyzer" output="screen">
    <param from="$(var config_file)"/>
    <param name="input_bag_path" value="$(var bag_path)"/>
    <param name="output_dir" value="$(var output_dir)"/>
  </node>
</launch>
